src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -220,26 +222,45 @@ function ComposerWithSuggestions({
                     debouncedUpdateFrequentlyUsedEmojis();
                 }
             }
-            const newCommentConverted = convertToLTRForComposer(newComment);
+
+            let newCommentConverted = newComment;
+            const prevComment = commentRef.current;
```

Comment:
Here I just put that in a variable, it could work but I prefer using the same pattern that was used (just to let everything be like before), e.g:

https://github.com/Expensify/App/blob/2feaac80198481546ee6aa630aa1c7b386ec5605/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js#L225


https://github.com/Expensify/App/blob/2feaac80198481546ee6aa630aa1c7b386ec5605/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js#L234